sdcm/nemesis.py

```diff
@@ -940,132 +942,66 @@ def disrupt_no_corrupt_repair(self):
 
-            for line in resharding_logs:
-                # Find all files that were created after resharding
-                for one_file in re.findall(r"(/var/.*?),", line, re.IGNORECASE):
-                    # The file path have to include "upload" folder
-                    assert '/upload/' in one_file, "Loaded file was resharded not in 'upload' folder"
+    # pylint: disable=too-many-statements
+    def disrupt_nodetool_refresh(self, big_sstable: bool = False):
+        # Checking the columns number of keyspace1.standard1
+        self.log.debug('Prepare keyspace1.standard1 if it does not exist')
+        self._prepare_test_table(ks='keyspace1', table='standard1')
+        column_num = SstableLoadUtils.calculate_columns_count_in_table(self.target_node)
+
+        # Note: when issue #6617 is fixed, we can try to load snapshot (cols=5) to a table (1 < cols < 5),
+        #       expect that refresh will fail (no serious db error).
+        if 1 < column_num < 5:
+            raise UnsupportedNemesis("Schema doesn't match the snapshot, not uploading")
+
+        test_data = SstableLoadUtils.get_load_test_data_inventory(column_num, big_sstable=big_sstable,
+                                                                  load_and_stream=False)
+
+        result = self.target_node.run_nodetool(sub_cmd="cfstats", args="keyspace1.standard1")
 
         if result is not None and result.exit_status == 0:
+            key = '0x32373131364f334f3830'
             # Check one special key before refresh, we will verify refresh by query in the end
             # Note: we can't DELETE the key before refresh, otherwise the old sstable won't be loaded
             #       TRUNCATE can be used the clean the table, but we can't do it for keyspace1.standard1
-            query_verify = "SELECT * FROM keyspace1.standard1 WHERE key=0x32373131364f334f3830"
+            query_verify = f"SELECT * FROM keyspace1.standard1 WHERE key={key}"
             result = self.target_node.run_cqlsh(query_verify)
             if '(0 rows)' in result.stdout:
-                self.log.debug('Key 0x32373131364f334f3830 does not exist before refresh')
+                self.log.debug('Key %s does not exist before refresh', key)
             else:
-                self.log.debug('Key 0x32373131364f334f3830 already exists before refresh')
+                self.log.debug('Key %s already exists before refresh', key)
 
             # Executing rolling refresh one by one
             for node in self.cluster.nodes:
-                system_log_follower = do_refresh(node)
-                validate_resharding_after_refresh(system_log_follower)
+                SstableLoadUtils.upload_sstables(node, test_data=test_data[0])
+                system_log_follower = SstableLoadUtils.run_refresh(node, test_data=test_data[0])
+                SstableLoadUtils.validate_resharding_after_refresh(system_log_follower, node)
```

Comment:
just small notice, 
you have method:   SstableLoadUtils.validate_load_and_stream_status(load_on_node, system_log_follower) where first param is node , second is log_follower

may by it is better to have same parameter order for this method too?

Reply:
> just small notice,
> you have method: SstableLoadUtils.validate_load_and_stream_status(load_on_node, system_log_follower) where first param is node , second is log_follower
> 
> may by it is better to have same parameter order for this method too?

@aleksbykov I think you confised between 2 methods: **validate_resharding_after_refresh** and **validate_load_and_stream_status**. It's different methods

Reply:
the method are different, but i think that order of parameters in method signature could be the same.

Reply:
done